packages/cli/internal/pkg/aws/cwl/stream_logs.go

```diff
@@ -38,6 +38,8 @@ func (c Client) StreamLogs(ctx context.Context, logGroupName string, streams ...
 			if aws.ToString(lastToken) != aws.ToString(output.NextToken) {
 				stream <- StreamEvent{Logs: parseEventLogs(output.Events, startTime)}
 				lastToken = output.NextToken
+			} else {
```

Comment:
Could you please explain this change?

Reply:
Currently, we don't show any messages whenever an empty log comes through which was a regression from a previous change. This is particularly evident when you look at the debug statement in the logs_core package as the statement never triggered.